### ISSUE TITLE
Are there any plans to support dates?

### ISSUE THREAD

**Issue by darrenparkinson** [spf13/pflag#217]
I notice that IP Addresses and Duration are supported options.  Are there any plans to support dates as flags? Thanks.

**Comment by therealmitchconnors** [comment]
How would you propose solving for different date formats and time zones?  These aren't impossible to solve, but it's important to lay out the specifics from the start.

**Comment by therealmitchconnors** [comment]
Yes, IPNet does have to be parsed, but there are clear RFCs for parsing them, and they are never ambiguous or locale dependent, AFAIK.  If you came to a good solution with Value{}, I'd love to see what you did!

**Comment by tomasaschan** [comment]
This issue has not been updated in a long time. Is it still relevant?

(If I receive no answer before the end of the summer, I'll close it. Feel free to re-open if you come here after that happens and it is relevant to you - but if you do, please also comment and specify what outcome you're interested in!)

**Comment by brandon1024** [comment]
I think this is safe to close. This has been supported since `v1.0.7` with `Time()` and related flag functions.
